modules/core/src/controllers/controller.js

```diff
@@ -423,4 +422,9 @@ export default class Controller {
     return this.updateViewport(newControllerState, LINEAR_TRANSITION_PROPS);
   }
   /* eslint-enable complexity */
+
+  _getTransitionControls(props) {
```

Comment:
This seems strangely specific.
Instead of `LINEAR_TRANSITION_PROPS`, can just call a method `_getLinearTransitionProps`?

Reply:
We have two flags that control keyboard/double tap transitions, this method can be more customizable (to enable, disable one or both transitions) by child controllers.

Reply:
The crash is caused by transition, not the interaction itself. Force disabling the interactions is excessive.